Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -132,14 +132,14 @@ PaymentMethodsViewModelInputs, PaymentMethodsViewModelOutputs {
     self.didTapAddCardButtonProperty.value = ()
   }
 
-  fileprivate let cardAddedSuccessfullyProperty = MutableProperty("")
-  public func cardAddedSuccessfully(_ message: String) {
-    self.cardAddedSuccessfullyProperty.value = message
+  fileprivate let addNewCardSucceededProperty = MutableProperty("")
```

Comment:
🤔 I see it was this way before but usually we'd not set a default string in the `MutableProperty` initializer, these defaults make it harder to track down where values are coming from.